pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -1009,6 +1009,7 @@ func (p *Provider) Create(
 			fmt.Sprintf("size=%dGB", providerOpts.PDVolumeSize),
 			"auto-delete=yes",
 		}
+		// TODO(pavelkalinnikov): set the "provisioned-throughput" in MB/s too.
 		args = append(args, "--create-disk", strings.Join(pdProps, ","))
```

Comment:
afaict, gce doesn't allow a user to tweak the throughput of an ssd, but maybe i'm looking in the wrong place? https://cloud.google.com/compute/docs/disks/performance#performance_limits

Reply:
I was under impression that this was slightly configurable for the block storage.

I noticed the `provisioned-throughput` option in the [documentation](https://cloud.google.com/sdk/gcloud/reference/compute/instances/create#--create-disk) for the `--create-disk` command:

```
provisioned-throughput
	Indicates how much throughput to provision for the disk. This sets the number of throughput mb per
	second that the disk can handle.
```

There is one in `compute disks create` [command](https://cloud.google.com/sdk/gcloud/reference/compute/disks/create#--provisioned-throughput) too, but it mentions that this works only for [Hyperdisk Throughput](https://cloud.google.com/compute/docs/disks/hyperdisks) disks. So maybe that is the extent to which one can tune throughput in GCE.

Reply:
Extended the TODO with a documentation link.

Reply:
makes sense. thanks for the explanation!